enclave/src/main/java/org/hyperledger/besu/enclave/Enclave.java

```diff
@@ -54,42 +56,72 @@ public boolean upCheck() {
     }
   }
 
-  public SendResponse send(final SendRequest content) {
+  public SendResponse sendLegacy(
+      final String payload, final String privateFrom, final List<String> privateFor) {
+    final SendRequestLegacy sendRequestLegacy =
+        new SendRequestLegacy(payload, privateFrom, privateFor);
     return post(
         JSON,
-        content,
+        sendRequestLegacy,
         "/send",
         (statusCode, body) -> handleJsonResponse(statusCode, body, SendResponse.class));
   }
 
-  public ReceiveResponse receive(final ReceiveRequest content) {
+  public SendResponse sendBesu(
+      final String payload, final String privateFrom, final String privacyGroupId) {
+    final SendRequestBesu sendRequestBesu =
+        new SendRequestBesu(payload, privateFrom, privacyGroupId);
+    return post(
+        JSON,
+        sendRequestBesu,
+        "/send",
+        (statusCode, body) -> handleJsonResponse(statusCode, body, SendResponse.class));
+  }
+
+  public ReceiveResponse receive(final String enclaveKey) {
+    final ReceiveRequest receiveRequest = new ReceiveRequest(enclaveKey);
+    return post(
+        ORION,
+        receiveRequest,
+        "/receive",
+        (statusCode, body) -> handleJsonResponse(statusCode, body, ReceiveResponse.class));
+  }
+
+  public ReceiveResponse receive(final String enclaveKey, final String to) {
+    final ReceiveRequest receiveRequest = new ReceiveRequest(enclaveKey, to);
     return post(
         ORION,
-        content,
+        receiveRequest,
         "/receive",
         (statusCode, body) -> handleJsonResponse(statusCode, body, ReceiveResponse.class));
   }
 
-  public PrivacyGroup createPrivacyGroup(final CreatePrivacyGroupRequest content) {
+  public PrivacyGroup createPrivacyGroup(
```

Comment:
would life be nicer if this tool a collection of addresses, then converted it to an array internally? (I _suspect_ not, as the json received is an array ... but ... seeing String[] on interfaces makes me wonder...